crates/hc_sandbox/src/save.rs

```diff
@@ -65,18 +64,23 @@ pub fn remove(hc_dir: PathBuf, existing: Existing) -> std::io::Result<usize> {
                         i,
                         path.display()
                     );
-                    to_remove_indices.push(i);
+                    if !to_remove_indices.contains(&i) {
+                        to_remove_indices.push(i);
```

Comment:
add only new indices

Reply:
As below, the user would have to have done `hc s remove 0 0` to make that happen and that's a user error that they can trace back to their own command.

Reply:
change removed